### PR TITLE
Fix environment with env_file bug

### LINKED ISSUE #1013: 'environment' with 'env-file' specified together for service in docker-compose.yml don't  work
In such case key-value pairs in 'environment' sections are omitted.

For example

```json
services:
  namenode:
    image: bde2020/hadoop-namenode:2.0.0-hadoop2.7.4-java8
    volumes:
      - namenode:/hadoop/dfs/name
    environment:
      - CLUSTER_NAME=test
    env_file:
      - ./hadoop-hive-namenode.env
    ports:
      - "50070:50070"
      - "8020:8020"
```

`CLUSTER_NAME` won't be set

### PR DESCRIPTION
**Pull request by hangyan**
Fix #1013

### CODE REVIEW

**Comment by hangyan** [comment]
@cdrage Do you have time to reivew #1020 #1007 for me? Not in a rush, but i can only continue to work after these PRs merged

**Comment by cdrage** [comment]
@hangyan Sorry, been busy with other projects. I'll review them now! :+1:

File: pkg/loader/compose/v3.go [hunk 1]
```diff
@@ -235,7 +235,7 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 	for _, composeServiceConfig := range composeObject.Services {
 
 		// Standard import
-		// No need to modify before importation
+		// No need to modify `before importation
```
Comment (cdrage) [inline pkg/loader/compose/v3.go]: remove random `

**Comment by cdrage** [comment]
Feel free to merge after the fix

**Pull request merged**